salon:
  qr_text: "ROOM=SALON"
  entry:
    x: -0.738440
    y: 1.117620
    z: -0.049833
    w: 0.998757
  cleaning_points:
    - {x:  -3.429173707962036, y: 4.883893966674805, z: 0.0, w: 1.0}
    - {x: -2.615974187850952, y: 0.945573091506958, z: 0.0, w: 1.0}
    - {x: -0.3982395529747009, y: 4.7019805908203125, z: 0.0, w: 1.0}

mutfak:
  qr_text: "ROOM=MUTFAK"
  entry:
    x: -4.145525
    y: 1.776980
    z: 0.999979
    w: 0.006466
  cleaning_points:
    - {x: -6.537049770355225, y: 4.531886100769043, z: 0.0, w: 1.0}
    - {x: -6.127171039581299, y: 1.609633445739746, z: 0.0, w: 1.0}
    - {x: -6.849677562713623, y: 3.320004463195801, z: 0.0, w: 1.0}

koridor:
  qr_text: "ROOM=KORIDOR"
  entry:
    x: 0.416697
    y: 0.273836
    z: 0.726211
    w: 0.687471
  cleaning_points:
    - {x: 0.8730320334434509, y: 0.21160617470741272, z: 0.0, w: 1.0}
    - {x: 0.9763827919960022, y: 0.35188573598861694, z: 0.0, w: 1.0}
    - {x: 2.1549835205078125, y: 0.3071087896823883, z: 0.0, w: 1.0}

yatak_odasi:
  qr_text: "ROOM=YATAKODASI"
  entry:
    x: 2.906681
    y: 0.643564
    z: -0.768950
    w: 0.639308
  cleaning_points:
    - {x: 4.893410682678223, y: 4.061635971069336, z: 0.0, w: 1.0}
    - {x: 6.652835369110107, y: 2.6393637657165527, z: 0.0, w: 1.0}
    - {x: 4.879083156585693, y: 0.91921466588974, z: 0.0, w: 1.0}
